subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/screenshot_test_report/ScreenshotTestImageLoader.kt

```diff
@@ -0,0 +1,55 @@
+import android.annotation.SuppressLint
+import com.avito.filestorage.FutureValue
+import com.avito.filestorage.RemoteStorage
+import com.avito.logger.Logger
+import java.io.File
+
+@SuppressLint("LogNotTimber")
+
+interface ScreenshotTestImageUploader {
+
+    fun uploadScreenshots(referenceImage: File?, generatedImage: File?): String
```

Comment:
Advice. Make the uploader simpler. With `referenceImage` == null it make no sence
So you could change `uploadScreenshots` to receive only non-null values

Reply:
done